c3s_eqc_automatic_quality_control/download.py

```diff
@@ -278,12 +278,11 @@ def split_request(
 def download_and_transform_chunk(
     collection_id: str,
     request: dict[str, Any],
-    func: None
+    transform_func: None
```

Comment:
OK with me. Why though?
If the issue is cacholote, I've fixed it a couple of days ago.

Reply:
It raises a `TypeError: hexdigestify_python_call() got multiple values for argument 'func'`. Good to know it's fixed on cacholote side. On top of that `transform_func` could be a clearer name. Explicit it's better than implicit they say, right?

Reply:
sounds good, I'll update the notebooks then